src/system/angband-version.cpp

```diff
@@ -1,7 +1,7 @@
 ﻿#include "system/angband-version.h"
 #include "system/angband.h"
 
-void put_version(char *buf)
+std::string get_version(void)
```

Comment:
Explicit 'void' is unnecessary.
If you write it in any other PR, please change them all.

Reply:
The latest forced push drops `void`.